frontend/src/utils/server-utils.ts

```diff
@@ -302,17 +302,15 @@ async function generateIntermediateDateItemFromDataFile(
 
 export function generateIntermediateDateItemFromValidity(layer: ValidityLayer) {
```

Comment:
This is not directly linked to this PR, but since we are modifying this function we could remove the usage of `lodash` functions `sortBy` and `sortedUniqBy`